content/kubermatic/main/installation/upgrading/upgrade-from-2.21-to-2.22/_index.en.md

```diff
@@ -0,0 +1,103 @@
+INFO[0072]    📦 Deploying Kubermatic Operator…
+INFO[0072]       Deploying Custom Resource Definitions…
+INFO[0078]       Migrating UserSSHKeys…
+INFO[0079]       Migrating Users…
+INFO[0079]       Migrating ExternalClusters…
+INFO[0079]       Deploying Helm chart…
+INFO[0079]       Updating release from 2.20.6 to 2.21.0…
+INFO[0136]    ✅ Success.
+INFO[0136]    📦 Deploying Telemetry
+INFO[0136]       Updating release from 2.20.6 to 2.21.0…
+INFO[0142]    ✅ Success.
+INFO[0142]    📡 Determining DNS settings…
+INFO[0142]       The main LoadBalancer is ready.
+INFO[0142]
+INFO[0142]         Service             : nginx-ingress-controller / nginx-ingress-controller
+INFO[0142]         Ingress via hostname: <AWS ELB Name>.eu-central-1.elb.amazonaws.com
+INFO[0142]
+INFO[0142]       Please ensure your DNS settings for "<Hostname>" include the following records:
+INFO[0142]
+INFO[0142]          <Hostname>    IN  CNAME  <AWS ELB Name>.eu-central-1.elb.amazonaws.com.
+INFO[0142]          *.<Hostname>  IN  CNAME  <AWS ELB Name>.eu-central-1.elb.amazonaws.com.
+INFO[0142]
+INFO[0142] 🛬 Installation completed successfully. Time for a break, maybe? ☺
+```
+
+Upgrading seed clusters is no longer necessary in KKP 2.22, unless you are running the `minio` Helm chart as distributed by KKP on them. Apart from upgrading the `minio` chart, no manual steps for seed clusters are required. They will be automatically upgraded by KKP components.
+
+You can follow the upgrade process by either supervising the pods on master and seed clusters (by simply checking `kubectl get pods -n kubermatic` frequently) or checking status information for the `Seed` objects. A possible command to extract the current status by seed would be:
+
+```sh
+$ kubectl get seeds -A -o jsonpath="{range .items[*]}{.metadata.name} - {.status}{'\n'}{end}"
+kubermatic - {"clusters":3,"conditions":{"KubeconfigValid":{"lastHeartbeatTime":"2022-08-03T10:10:32Z","reason":"KubeconfigValid","status":"True"},"ResourcesReconciled":{"lastHeartbeatTime":"2022-08-25T09:30:52Z","lastTransitionTime":"2022-08-25T09:30:52Z","reason":"ReconcilingSuccess","status":"True"}},"phase":"Healthy","versions":{"cluster":"v1.23.6","kubermatic":"v2.21.0"}}
+```
+
+Of particular interest to the upgrade process is if the `ResourcesReconciled` condition succeeded and if the `versions.kubermatic` field is showing the target KKP version. If this is not the case yet, the upgrade is still in flight. If the upgrade is stuck, try `kubectl -n kubermatic describe seed <seed name>` to see what exactly is keeping the KKP Operator from updating the seed cluster.
+
+After a Seed was successfully upgraded, user clusters on that Seed should start updating. Observe their control plane components in the respective cluster namespaces if you want to follow the upgrade process. This is the last step of the upgrade, after all user clusters have completed component rotation the KKP upgrade is complete.
+
+## Post-Upgrade Considerations
+
```

Comment:
I guess KubeVirt migration doc should land here, right? I will add it once the PR is merged.

Reply:
It would make sense to have them here, but we can also link to them. Whatever works for you. If you want to add them to this document, feel free to push the changes to my branch, so you don't have to wait.

Reply:
Ok then I will link to them. I plan to open my own PR with KubeVirt migration and later I will put a link in this section.

Reply:
I've linked to the KubeVirt migration docs now.